### ISSUE TITLE
Poor Predicting Accuracy

### ISSUE THREAD

**Issue by Charlie818** [hx173149/C3D-tensorflow#39]
I used the  **/c3d_ucf101_finetune_whole_iter_20000_TF.model**  and followed the guidance to generate the test list on UCF101. Then, I only changed  the gpu_num into 1 to match my computer and ran the predict_c3d_ucf101.py. However, the accuracy is terrible. I got **1.3%** accuracy (32/3261).

**Comment by geroge-gao** [comment]
have you ever used the c3d_ucf101_finetune_whole_iter_20000_TF.model for fine-tuning. when I use the pre-trained model for training, my train accuracy is almost 100%，but the test accuracy is not really good.
![1ir 223 sf4s 7p7s9 _spf](https://user-images.githubusercontent.com/11483800/40108948-e11b3a7a-592e-11e8-89ff-09baefa0f902.png)
![sn5x 3liceho_4 slni9d](https://user-images.githubusercontent.com/11483800/40108955-e7667552-592e-11e8-9a76-7bba65e69623.png)

**Comment by Charlie818** [comment]
Did you comment the line that transposes the output of conv layers?

**Comment by geroge-gao** [comment]
yes. but it doesn't work.

**Comment by geroge-gao** [comment]
yes,i did as it mentioned, but it did't work.
![9 q1yif 1 f ng4eutcyl](https://user-images.githubusercontent.com/11483800/40118273-0a331992-594c-11e8-856d-016ca1403997.png)
but when I used the  sports1m_finetuning_ucf101.model.the test accuracy can be 70%.

**Comment by Charlie818** [comment]
According to my implementation, given a simple SVM classifier, it reaches 73.6% accuracy in UCF-101 . You could reference it in my repo [C3D Model](https://github.com/Charlie818/VideoAnalysis/blob/master/video/c3d_model.py).

If you find it helpful, please remember to star it. :）

**Comment by geroge-gao** [comment]
in your code I see you comment the tranpose operation, could you tell me what role the operation is?because I don't see the operation in other project of C3D

**Comment by Charlie818** [comment]
The operation simply transposes the output matrix. Taking a 2d matrix for example, it converts  [[1,2],[3,4]] into [[1,3],[2,4]]

**Comment by geroge-gao** [comment]
yeah,I know that ,but what i want't to ask is why there should be a tranpose operation? you can see[c3d-keras](https://github.com/axon-research/c3d-keras/blob/master/c3d_model.py). there is no tranpose operation.

**Comment by kasparov92** [comment]
What if I would like to continue training on the model 'sports1m_finetuning_ucf101.model', shall I use or not the transpose ?

**Comment by jiarongqiu** [comment]
As the author mentioned in the fifth note of [usages](https://github.com/hx173149/C3D-tensorflow/blob/master/README.md/#usage), you should use the transpose operation.
